docs/dev-environment.md

```diff
@@ -88,7 +94,23 @@ Notable records created via seed script:
 
 ## Backend services
 
-The application integrates with multiple backend services which are required to be available for most HTTP request to complete successfully.
+The application integrates with multiple backend services:
+
+## Worker
+
+Worker processes (`pbworker`) are responsible for running background jobs. There must be one or more processes running in order to pick up background jobs (e.g. launch reservations). There are multiple configuration options available via `WORKER_QUEUE`:
+
+* `redis` - uses queue via Redis
+* `postgres` - uses jobqueue implementation for the queue
+* `memory` - in-memory worker (default option)
+
+The default behavior is the in-memory worker, which spawns a single goroutine within the main apllication which picks up all jobs sequentially. This is only meant for development setups so that no extra worker process is required when testing backround jobs.
```

Comment:
applic
```suggestion
The default behavior is the in-memory worker, which spawns a single goroutine within the main application which picks up all jobs sequentially. This is only meant for development setups so that no extra worker process is required when testing background jobs.
```